aries-site/src/data/structures/components.js

```diff
@@ -344,15 +366,15 @@ export const components = [
       background: 'background-front',
     },
     relatedContent: ['TextInput', 'Header'],
-    tags: ['search input'],
+    tags: ['search input', 'search accessibility standards', 'auto-suggest in search', 'search suggestions', 'search placement', 'search icon'],
   },
   {
     name: 'Select',
     category: 'Inputs',
     description:
-      'Flexible input allowing users to choose from a list of options.',
+      'Select is a flexible input field that allows users to choose from a list of options. Also known as a drop-down field.',
```

Comment:
```suggestion
      'Select is a flexible input field that allows users to choose from a list of options.',
```
Select aligns in naming with the native HTML component, so I'm proposing we don't introduce other names to call it.

Reply:
agree `select` is more widely known